src/components/BooksForm/BooksForm.jsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react';
+import uuid from 'uuid';
+
+export default () => {
```

Comment:
moving forward, this should be a class components so we can hold state of the form

Reply:
Resolved the above mentioned issue and created an Stateful component, which for the time being it holds the categories array.